src/services/dealer-price/proto/services/price/v1/price_service.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
```

Comment:
could we merge this one first? 

https://github.com/GaloyMoney/galoy/pull/1429/files

to get a smaller diff

Reply:
done